squad/core/models.py

```diff
@@ -761,6 +760,12 @@ class Test(models.Model):
     def __str__(self):
         return self.name
 
+    @property
+    def name(self):
+        if self.metadata is not None:
+            return self.metadata.name
+        return 'missing test name'
```

Comment:
it's probably a matter of taste, but I would prefer 'missing test name` to be the fallback and not the default, i.e.

```python
if self.metadata is None:
    return 'missing test name'
return self.metadata.name
```

Reply:
Makes sense, I think yours is better to look at :)